components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1463,6 +1474,60 @@ def get_thumb_url(iid):
 
 @login_required()
 @jsonp
+def get_thumbnails_json(request, w=None, conn=None, **kwargs):
+    """
+    Returns base64 encoded jpeg with the rendered thumbnail for images
+    'id'
+
+    @param request:     http request
+    @param w:           Thumbnail max width. 96 by default
+    @return:            http response containing base64 encoded thumbnails
+    """
+    if w is None:
+        w = 96
+    image_ids = get_longs(request, 'id')
+    logger.debug("Image ids: %r" % image_ids)
+    if len(image_ids) > settings.THUMBNAILS_BATCH:
+        return HttpJavascriptResponseServerError(
+            'Max %s thumbnails at a time.' % settings.THUMBNAILS_BATCH)
```

Comment:
Well the doc says regarding full set of changes from https://github.com/openmicroscopy/openmicroscopy/pull/5062 including client changes. If we are not happy of that rebase I can close this PR